managed-token/sdk/.crates/.crates2.json

```diff
@@ -0,0 +1 @@
+{"installs":{"shank-cli 0.0.5 (registry+https://github.com/rust-lang/crates.io-index)":{"version_req":"^0.0.5","bins":["shank"],"features":[],"all_features":false,"no_default_features":false,"profile":"release","target":"aarch64-apple-darwin","rustc":"rustc 1.60.0 (7737e0b5c 2022-04-04)\nbinary: rustc\ncommit-hash: 7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c\ncommit-date: 2022-04-04\nhost: aarch64-apple-darwin\nrelease: 1.60.0\nLLVM version: 14.0.0\n"}}}
```

Comment:
Same with this

Reply:
@thlorenz probably knows more about what gets autogenerated. I think this can probably get deleted, but I don't think it really matters that much?

Reply:
I'll remove it thought